modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
```

Comment:
Nit-est of nits:
```suggestion
// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
```
A few others like this for new files if you want to add for consistency.

Reply:
Thank you. Fixed in the modules affected by this PR.